app/gateway/entities/routes/expressions.md

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
based on this url, would you expect `/gateway/entities/routes/` to be the current route entity reference page?